cmp/resizable/impl/Dragger.js

```diff
@@ -11,12 +11,16 @@ import {elemFactory, hoistComponent} from 'hoist/core';
 import {div} from 'hoist/layout';
 import './Dragger.scss';
 
+/** This is an implementation class private to Hoist
+ * @private
+ */
 @hoistComponent()
 export class Dragger extends Component {
 
     resizeState = null;
 
     static propTypes = {
+        /** Position of resize border in relation to resizable component */
```

Comment:
probably ok to keep the prop types on this impl class since they are already here, but ditch the obvious comment

Reply:
Maybe this was dense of me, but when I first glanced at this class (without reading any further than the prop) I thought the 'side' could have meant the side of the viewport we would be dragging the component out from. i.e. top wouldn't mean the drag bar is on top of the cmp, but that you'd drag down from the top of the window.